React/prop-types.md

```diff
@@ -0,0 +1,82 @@
+# Prop Types
+
+## Learning Goals
+By the end of this lesson you should be able to:
+
+- Explain the use of `PropTypes`
+- Use `PropTypes` to define and document the props a component can take
+
+You may have noticed this message from your JavaScript linter.  "____ is missing from props validation (react/prop-types)."  The linter is encouraging you to use a feature of React called Prop-types in which you list the props a component takes and their types.
+
+![prop types error](images/prop-types.png)
+
+In this lesson we will define `propTypes` in our React components which will provide some type-checking and documentation of our React component props.
+
+## Student Component Example
+
+Our Student component takes a name & an e-mail as props.  We can define the properties our `Student` component takes by adding a `static` `propTypes` attribute.
+
+First you need to `import PropTypes from 'prop-types';` to import the package.  Then add a static `propTypes` attribute to your class like below.
```

Comment:
I feel like there is not quite enough context about where exactly this should go in the component class. Could you add that detail?